pkg/apis/pipeline/v1/task_validation_test.go

```diff
@@ -2254,3 +2254,86 @@ func TestGetArrayIndexParamRefs(t *testing.T) {
 		})
 	}
 }
+
+func TestParamEnum(t *testing.T) {
```

Comment:
better separate this to success and failure like tests in pipeline validation

Reply:
I've splitted the tests, PTAL!